keras_nlp/models/bert.py

```diff
@@ -285,30 +305,58 @@ def BertBase(name=None, trainable=True):
     or classification task networks.
 
     Args:
+        weights: String, optional. Name of pretrained model to load weights.
+            Should be one of {names}.
+            If None, model is randomly initialized. Either `weights` or
+            `vocabularly_size` must be specified, but not both.
+        vocabulary_size: Int, optional. The size of the token vocabulary. Either
+            `weights` or `vocabularly_size` must be specified, but not both.
         name: String, optional. Name of the model.
         trainable: Boolean, optional. If the model's variables should be
             trainable.
 
     Example usage:
     ```python
     # Randomly initialized BertBase encoder
-    encoder = keras_nlp.models.BertBase()
+    encoder = keras_nlp.models.BertBase(vocabulary_size=10000)
```

Comment:
I think you would need to format all the BertBase references in the docstring right?

Reply:
Copied all the code snippets from the test file to get the black formatting. Obviously they run as intended if the tests pass!

Reply:
I think you may have misunderstood this comment?

You are adding a format string so this could work with say BertLarge right? But this has the string literal BertBase still hard coded. I think you need to format string `type` in here for this to actually render correctly for other bert types.

Reply:
Ah yes I understand now. I decided to leave that out until we had more arches since it adds a lot of cruft (including splitting off the "example usage" block which is different for every model).

Reply:
BTW, `BASE_DOCSTRING` is quite the play on words here, eh eh? Guess we can take it literally for now. 🤡🤡🤡

Reply:
Got it! That sounds good to me.

And yeah "base" like base class and "base" like medium sized model is a really confusing naming collision. But oh well, bert-base is definitely a well known moniker.